Chirp/Pages/MorePage/MorePageViewController.swift

```diff
@@ -75,4 +75,13 @@ extension MorePageViewController: UITableViewDelegate, UITableViewDataSource {
         cell.cellImage.image = UIImage(named: name)
         return cell
     }
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let rows: [CellNames] = [.pushNotification, .contributors, .logout]
```

Comment:
Could you move this to the top of the class instead and update your datasource so that you now do rows.count, rows[indexPath.row], etc instead of always creating all arrays before comparing the index?